### ISSUE TITLE
Push rejected, no Cedar-supported app detected

### ISSUE THREAD

**Issue by kawashimaken** [DaftMonk/generator-angular-fullstack#1449]
I have the following experience:
FAILED CASE(now):
I also experiencing the same issue when I try to deploy my app to heroku. 
generator-angular-fullstack v2.1.1+ node v5.0.0

SUCCEEDED CASE(past):
I once succeeded in deploying another app to heroku with v2.1.1, + node 0.12.x) 

I am not sure if it has something to do with the node version.
now, when I run 
`yo angular-fullstack:heroku`
It can not proceed successfully. It returns:
`Push rejected, no Cedar-supported app detected `
My guess is that heroku side has had some updates.
I also tried a fresh ` yo angular-fullstack myapp `, but get the same result.

Any suggestions about which part I should dig deeper? Thanks in advance.

(I also posted in https://github.com/DaftMonk/fullstack-demo/issues/27, please let me know if I should remove one of them.)
Great project! thanks!

**Comment by eh-am** [comment]
I have successfully deployed my app using
```
grunt build
grunt buildcontrol:heroku
```

**Comment by kawashimaken** [comment]
@eh-am Thank you for your response, I did a fresh install again, when I run 
grunt build, I get some error about imagemin, I did the following
```
npm install grunt-contrib-imagemin
npm install optipng-bin
```
now I can run ```grunt build```, then ```grunt buildcontrol:heroku```
after I set up mongodb using ```heroku addons:create mongolab```
repeat the above, then it worked! 
Since in my failed one, I already have done a lot of modification, I will try ```grunt build``` first there, see if I can get any hint from there, thank you again.

**Comment by kawashimaken** [comment]
now, I can successfully init heroku by running 
```
grunt build
grunt buildcontrol:heroku
```
without the error I mentioned above.
I guess the reason is that I was trying to init heroku without fixing some module dependency errors, so this topic can be closed.
However I have got the problem as mentioned here 
https://github.com/DaftMonk/generator-angular-fullstack/issues/152#issuecomment-161985312
please give it a look, any suggestions are welcome.  Thank you in advance.

**Comment by Awk34** [comment]
@kawashimaken do you have the same issue using 3.0.1?

**Comment by kawashimaken** [comment]
Hi, @Awk34 thank you for your response.
Actually I haven't tried 3.0.1 yet.
What I am using is 2.1.1
And when I do a fresh generation use the generator, I had no problem deploying to heroku.
while the app I am working on dose fail due to the above problem.
since I am working on this new project, I added many directives, routes, endpoints, so things get much more complicated, still trying to figure out  which part is possibly wrong, but I don't think I touched any default express configuration. 
Any insight will be appreciated.
